app/Controllers/Http/MatchController.js

```diff
@@ -1,16 +1,29 @@
 'use strict';
 
+const moment = use('moment');
+
 const Match = use('App/Models/Match');
-class MatchController {
 
+const startOfToday = moment().startOf('day').format('YYYY-MM-DD HH:mm:ss');
+
+const matchTaken = async timeId => {
+  const matches = await Match.query()
+    .where('created_at', '>', startOfToday)
```

Comment:
It would be nice to have a query scope for this where clause because in the response we have the same where, check the documentation how to create a query scope.